test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/FunctionApps/powershell/EventHub/host.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
+      }
+    }
+  },
+  "managedDependency": {
```

Comment:
Why it's needed only for powershell apps ??

Reply:
[Reference on ManagedDependency](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/azure-functions/functions-host-json.md#manageddependency). Removed as not required.